sdk/ai/azure-ai-projects/tests/agents/test_agents_crud.py

```diff
@@ -15,7 +14,7 @@
 class TestAgentCrud(TestBase):
 
     @servicePreparer()
-    @recorded_by_proxy
+    @recorded_by_proxy()
```

Comment:
I added `()` here (even though not needed) just to test this case... Other tests continue to use the decorator without brackets.